k8s/crds/agents.x-k8s.io_sandboxes.yaml

```diff
@@ -523,6 +523,23 @@ spec:
                                         - fieldPath
                                         type: object
                                         x-kubernetes-map-type: atomic
+                                      fileKeyRef:
```

Comment:
nit: This change isn't just a folder rename, but regeneration of CRD. Suggest updating the commit message to make it more clear

Reply:
done